spec/lead_spec.rb

```diff
@@ -41,4 +41,33 @@
     specify { is_expected.to include(status_id: 2) }
     specify { is_expected.to include(tags: 'test lead') }
   end
+
+  describe "#contacts" do
+    let(:lead) { described_class.new(id: 2) }
+
+    it "fails if not persisted" do
+      expect { described_class.new.contacts }
+        .to raise_error(Amorail::Entity::NotPersisted)
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.